plugins/modules/files/ini_file.py

```diff
@@ -47,7 +47,18 @@
     description:
       - The string value to be associated with an I(option).
       - May be omitted when removing an I(option).
+      - Mutually exclusive with C(values).
```

Comment:
```suggestion
      - Mutually exclusive with I(values).
```
See https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation